charts/cf-runtime/values.yaml

```diff
@@ -536,6 +536,8 @@ runtime:
       METRICS_PROMETHEUS_HOST: '0.0.0.0'
       # -- Port for Prometheus metrics server
       METRICS_PROMETHEUS_PORT: 9100
+      # -- Trusted QEMU images used for docker builds - when left blank only 'tonistiigi/binfmt' is trusted.
```

Comment:
wouldn't it be better to just put `tonistiigi/binfmt` as default value here?

Reply:
Maybe. Are there other ways that the runtime can be deployed? I mean instead of using helm? In those cases removing the hard-coded default from the code and putting it here as a default instead would be a breaking change. And I didn't want to set the default here leading people to assume that if they set it to empty, that it would effectively stop the default from being accepted.

Reply:
I'm not aware of any other way. As an operator I prefer explicit configs over some source code magic. And as a safeguard I'm believer in "it's better to crash that produce unexpected result", so just fail pre-flight check on startup in case someone managed to launch it with empty list.

But that's just my opinion, waiting for someone else to chime in.

Reply:
That's a fair opinion and I tend to agree with you. I had a bit of a closer look and it seems as though there are still customers using the CLI to install their runners in which case removing the default from the code would be a breaking change unless I also update the CLI to provide the default. In this case I think to make things easier I'll keep it as is and once the CLI is no longer used we can tidy this up.